src/pre_process/m_initial_condition.fpp

```diff
@@ -206,6 +206,10 @@ contains
                 elseif (patch_ib(i)%geometry == 11) then
                     call s_3D_airfoil(i, ib_markers%sf, q_prim_vf, .true.)
```

Comment:
see above comment about putting a raw boolean in the middle of the argument list.